GOVERNANCE.md

```diff
@@ -0,0 +1,121 @@
+# Security Working Group
+
+The Node.js Security Working Group project is jointly governed by a Working
+Group (WG) that is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+- Technical direction
+- Project governance and process (including this policy)
+- Contribution policy
+- GitHub repository hosting
+- Conduct guidelines
+- Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project README.md.
+
+## Collaborators
+
+The security GitHub repository is maintained by the WG and additional
+Collaborators who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made Collaborators
+and given commit-access to the project. These individuals are identified by the
+WG and their addition as Collaborators is discussed during the periodic WG
+meeting.
+
+Note: If you make a significant contribution and are not considered for
+commit-access log an issue or contact a WG member directly and it will be
+brought up in the next WG meeting.
+
+Modifications of the contents of the security repository are made on a
+collaborative basis. Anybody with a GitHub account may propose a modification
+via pull request and it will be considered by the project Collaborators. All
+pull requests must be reviewed and accepted by a Collaborator with sufficient
+expertise who is able to take full responsibility for the change. In the case of
+pull requests proposed by an existing Collaborator, an additional Collaborator
+is required for sign-off. Consensus should be sought if additional Collaborators
+participate and there is disagreement around a particular modification. See
+Consensus Seeking Process below for further detail on the consensus model used
+for governance.
+
+Collaborators may opt to elevate significant or controversial modifications, or
+modifications that have not found consensus to the WG for discussion by
+assigning the WG-agenda tag to a pull request or issue. The WG should serve as
+the final arbiter where required.
+
+For the current list of Collaborators, see the project README.md.
+
+## WG Membership
+
+WG seats are not time-limited. There is no fixed size of the WG. However, the
+expected target is between 6 and 12, to ensure adequate coverage of important
+areas of expertise, balanced with the ability to make decisions efficiently.
+
+There is no specific set of requirements or qualifications for WG membership
+beyond these rules.
+
+The WG may add additional members to the WG by unanimous consensus.
```

Comment:
There is no link that is viewable, except by members, teams are private. Also, did you notice I listed all the members in the README?

There is no distinction between member and Collaborator in any of the WGs that I am aware of.